README.md

```diff
@@ -368,6 +368,23 @@ var options = {
 }
 ```
 
+
+##### Checkbox field
+
+To specify a checkbox field use: `type: "checkbox"`
+The `prefill` value can determine the default state of the checkbox and it is required.
+
+```js
+var options = {
+  additionalSignUpFields: [{
+    type: "checkbox",
+    name: "newsletter",
+    prefill: "true",
```

Comment:
I wanted to keep the modifications as small as possible.
The prefill value is needed otherwise the checkbox will be "invalid" in the store and the registration won't go through. 
Prefill must be string: https://github.com/auth0/lock/blob/master/src/connection/database/index.js#L107
I could change that, but it would need a lot of other changes elsewhere in the code